skops/card/_model_card.py

```diff
@@ -242,9 +243,12 @@ def select(self, key: str) -> Section:
             section = section.subsections[section_name]
         return section
 
-    def format(self) -> str:
+    def _format(self) -> str:
```

Comment:
Not quite sure why we need a separate `_format` method here?

It _could_ make sense to have a split like you suggest here _if_ we change `PlotSection` and `TableSection` to use `_format` too. This could be a good idea because we can move the `wrap_as_details` call to the parent class, avoiding each subclass having to call it. In general, this is a good idea, as it leaves less room for error.

There is a small issue with that suggestion, though. Right now, the `PlotSection` and `TableSection`, when folded, don't fold the whole output, but only the plot/figure, whereas the description stays as is. I think there can be an argument for changing that and argument for keeping it, I'm not sure. @adrinjalali what is your opinion?

Reply:
I think It's okay to have it in one method instead of introducing a private one, and also the whole thing folded instead of just the plot / table. The title should be indicative enough.